assets/js/constants.js

```diff
@@ -0,0 +1,4 @@
+export const ENDPOINTS = {
+	root: '/wc/blocks',
+	products: '/wc/blocks/products',
+};
```

Comment:
Looks great!  I know this is following my initial suggestion but I'm wondering if we should do something like this instead:

```js
const NAMESPACE = '/wc/blocks';
export const ENDPOINTS = {
    root: NAMESPACE,
    products: `${ NAMESPACE }/products`,
};
```

- reduces chance of typo errors for the namespace on future endpoints added.
- easier swapping out of namespace (albeit search and replace is arguably just as easy).

Reply:
Done in f7274821c361af6e22b728b73e7d75e371e8dafb.